src/main/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceImpl.java

```diff
@@ -139,7 +139,8 @@ private void updateHearingAndStatus(Long hearingId, ErrorDetails errorDetails) {
                 .sendMessage(objectMapperService.convertObjectToJsonNode(hmcHearingResponse).toString(),
                              hmcHearingResponse.getHmctsServiceCode(),hearingId.toString());
             if (hmcHearingResponse.getHearingUpdate().getHmcStatus().equals(HearingStatus.EXCEPTION.name())) {
-                log.error("Hearing id: " +  hearingId + " updated to status Exception");
+                log.info("Hearing id: " + hearingId + "has response of type :" + MessageType.ERROR);
```

Comment:
Let's combine both the logs into one error log
Ex: log.error("Hearing id: " + hearingId + " has response of type : " + MessageType.ERROR + " updated to status Exception");

Reply:
Sorry Rajaram but we can't do that -  `log.error("Hearing id: " +  hearingId + " updated to status Exception");` is picked up by dynatrace to raise an incident ticket. Combining both statements will break the regex they look for

Reply:
@RebeccaBaker,  Rajaram added comments in  the Jira https://tools.hmcts.net/jira/browse/HMAN-560 to address the dynatrace reg ex